jobs/scripts/nightly-builds/nightly-builds.sh

```diff
@@ -11,8 +11,13 @@ set -e
 
 BUILDREQUIRES="libaio-devel libattr-devel libxml2-devel readline-devel openssl-devel fuse-devel glib2-devel userspace-rcu-devel libacl-devel libuuid-devel gperftools-devel gperftools-libs"
 
-if [ "${CENTOS_VERSION}" -eq 8 ]
-then
+if [ "${CENTOS_VERSION}" == "8-stream" ]; then
+    ENABLE_REPOS="--enablerepo=powertools"
+    BUILDREQUIRES="${BUILDREQUIRES} python3-devel rpcgen libtirpc-devel liburing-devel rsync "
+    yum -y update
+    yum -y install epel-release
+    cat /etc/os-release
+elif [ "${CENTOS_VERSION}" -eq 8 ]; then
```

Comment:
CentOS 8 does not exist anymore, not sure there is a benefit of keeping this

Reply:
Yeah! I can go ahead and remove that part.